src/routes/safe/components/AddressBook/index.tsx

```diff
@@ -149,11 +149,7 @@ const AddressBookTable = (): ReactElement => {
 
   const importEntryModalHandler = (addressList: AddressBookEntry[]) => {
     addressList.forEach((entry) => {
-      const checksumEntries = {
-        ...entry,
-        address: checksumAddress(entry.address),
-      }
-      dispatch(addressBookAddOrUpdate(makeAddressBookEntry(checksumEntries)))
+      dispatch(addressBookAddOrUpdate(makeAddressBookEntry(entry)))
```

Comment:
Not related to this PR but why are we importing the entries one by one?

Reply:
because at the time of the implementation @juampibermani didn't have the import action available.

that will change as part of #2317